nixos/modules/security/pam.nix

```diff
@@ -36,14 +36,61 @@ let
         '';
       };
 
-      u2fAuth = mkOption {
-        default = config.security.pam.enableU2F;
-        type = types.bool;
-        description = ''
-          If set, users listed in
-          <filename>~/.yubico/u2f_keys</filename> are able to log in
-          with the associated U2F key.
-        '';
+      U2F = {
```

Comment:
True.
Is that good occasion to move that decision to mailing list or ticket and make some naming convention agreements? I saw one ticket about that but as far as I remember it stucked.